tests/unit/test_client.py

```diff
@@ -61,6 +61,9 @@ def test_create_run(client_mocker, experiment_name='exp', run_name='run'):
         experiment_name=experiment_name,
         run_name=run_name,
         run_config=config,
+        device='cpu',
+        cpus=1,
+        gpus=1,
```

Comment:
Can you also check that they were sent correctly? i.e. `assert sent_request['json_data'][GPUS] == gpus`